rbac/server/api/roles.py

```diff
@@ -296,6 +469,23 @@
 
 
 @ROLES_BP.post("api/roles/<role_id>/members")
+@doc.summary("Add a member to a role.")
+@doc.description("Add a member to a role.")
+@doc.consumes(doc.JsonBody({"id": str}), location="body", required=True)
```

Comment:
doc.consumes() also takes in metadata and reason field.

Reply:
but the only required field is ID, how do we handle that?

Reply:
Let me look and see if we can do that.

Reply:
After an offline discussion we've decided we're going to list all fields in a required request body with a description denoting which fields are optional or required. This isn't ideal but it's functional and we'll do this until the `sanic-openapi` package allows us to individually list request body parameters as required or optional.